multi-agent/lib/mas/resources/builtin_service.py

```
@@ -0,0 +1,366 @@
+        rid: str,
+        identity: Identity,
+    ) -> Dict[str, Any] | None:
+        """Return the user's current overlay for a built-in resource, or None.
+
+        Decrypts sensitive fields so the UI can display masked values.
+        Only returns fields that are marked as user-configurable.
+        Draft built-ins are not configurable by end users.
+        """
+        resource = self._store.get(rid)
+        if resource.ownership != ResourceOwnership.BUILTIN:
+            raise ValueError("Resource is not a built-in resource")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)